src/eq_schema/Block.js

```diff
@@ -1,18 +1,17 @@
 const Question = require("./Question");
 const { get } = require("lodash");
 const { getInnerHTML } = require("../utils/HTMLUtils");
-const convertPipes = require("../utils/convertPipes");
 
 const pageTypeMappings = {
   QuestionPage: "Questionnaire",
   InterstitialPage: "Interstitial"
 };
 
 class Block {
-  constructor(title, page) {
+  constructor(title, description, page) {
     this.id = "block-" + page.id.toString();
     this.title = getInnerHTML(title);
-    this.description = getInnerHTML(convertPipes(page.description));
```

Comment:
Wouldn't we still want to convert piped values for page titles?

Reply:
Yes, but that's a separate bug, i think. Not related to this change?

Reply:
Okay, fair enough 👍

Reply:
Hmm actually need to investigate, seems piping isn't working at all now :-/